ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -329,6 +358,7 @@ class FamilyReads extends React.PureComponent {
             igvSampleIndividuals={igvSampleIndividuals}
             sampleTypes={sampleTypes}
             rnaReferences={rnaReferences}
+            options={[{ type: JUNCTION_TYPE, field: 'minJunctionEndsVisible', value: minJunctionEndsVisible }]}
```

Comment:
This is a premature optimization, and it will immediately conflict with react-perf linting rules I reenable in my open PR. Revert this back to `minJunctionEndsVisible` being a direct prop